src/transformers/models/swin/modeling_swin.py

```diff
@@ -927,6 +975,8 @@ def _init_weights(self, module):
         output_hidden_states (`bool`, *optional*):
             Whether or not to return the hidden states of all layers. See `hidden_states` under returned tensors for
             more detail.
+        interpolate_pos_encoding (`bool`, *optional*):
```

Comment:
We really only want this to be a boolean, we don't need a null value 

```suggestion
        interpolate_pos_encoding (`bool`, *optional*, defaults to `False`):
```